package-lock.json

```diff
@@ -1,385 +1,447 @@
 {
   "name": "calendar-backend",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "calendar-backend",
-      "version": "0.1.0",
+      "version": "0.1.1",
```

Comment:
Do we have any sort of guidelines for when to increase the version number?

Reply:
I don't think so, but we should figure that out. I felt there were enough changes to change the version number, but we should standardize that!

Reply:
This is something that will be part of the developer guidelines doc! I think the convention we should follow is semantic versioning: https://semver.org/